### PR TITLE
488 amino ID

### PR DESCRIPTION
**Pull request by RL0109**

### CODE REVIEW

**Comment by RL0109** [comment]
Closes #488

**Review by cpitman-11 — APPROVED** [review]

File: VMD_scripts/Blob_GUI.tcl [hunk 1]
```diff
@@ -202,34 +202,26 @@ proc graphRepUser2 {MolID Lmin H} {
 	set count 0
 	set sel [atomselect $MolID protein]
 	set user2length [lsort -unique [$sel get user2]]
-	$sel delete
+	
+	$sel delete 
+	set colorNum [list 23 24 10 21 22]
```
Comment (EzryStIago) [inline VMD_scripts/Blob_GUI.tcl:207]: What does this line do?

Comment (EzryStIago) [reply VMD_scripts/Blob_GUI.tcl:207]: That is, what is colorNum for?

File: VMD_scripts/blobulation.tcl [hunk 2]
```diff
@@ -222,12 +222,22 @@ proc hydropathyScores { hydropathyList Sequence } {
...
 	foreach amino $Sequence {
 		if {[lsearch -exact $hydropathyList $amino] == -1} {
-			puts "Unkown amino acid ending program"
-			return -1
-		}
-		
+			if {$amino == "HID" || $amino == "HIE"} {
+				set value [dict get $hydropathyList "HIS"]
+			} else {
+				set aminoList {}
```
Comment (EzryStIago) [inline VMD_scripts/blobulation.tcl:228]: aminoList is the list of unrecognized amino acids, right? Can you rename this to something like unknownResnames?

**Review by EzryStIago — CHANGES_REQUESTED** [review]
Comment on readability: please document your changes to Blob_GUI.tcl - they don't seem to be related to the stated goal of closing issue 488. Both inline and in the conversation of this PR.

File: VMD_scripts/blobulation.tcl (hunk 2 above)
Comment (RL0109) [reply VMD_scripts/blobulation.tcl:228]: Done

**Review by EzryStIago — APPROVED** [review]

**Pull request merged**